src/hexo/themes/documentation/source/images/results-docs-icon.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24.19 28.41"><defs><style>.cls-1{fill:#4046dd;}</style></defs><title>docs-icon</title><path class="cls-1" d="M23.51 28.41H.68a.68.68 0 0 1-.68-.68V.68A.68.68 0 0 1 .68 0h22.83a.68.68 0 0 1 .68.68v27.05a.68.68 0 0 1-.68.68zM1.36 27.05h21.47V1.36H1.36z"/><path class="cls-1" d="M18 8.06H6A.68.68 0 1 1 6 6.7h12a.68.68 0 1 1 0 1.36zM18 12.75H6a.68.68 0 0 1 0-1.36h12a.68.68 0 0 1 0 1.36zM18 17.44H6a.68.68 0 0 1 0-1.36h12a.68.68 0 1 1 0 1.36zM18 22.13H6a.68.68 0 0 1 0-1.36h12a.68.68 0 0 1 0 1.36z"/></svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24.2 28.4"><style>.a{fill:#4046dd}</style><path d="M23.51 28.41H.68a.68.68 0 0 1-.68-.68V.68A.68.68 0 0 1 .68 0h22.83a.68.68 0 0 1 .68.68v27.05a.68.68 0 0 1-.68.68zM1.36 27.05h21.47V1.36H1.36z" class="a"/><path d="M18 8.06H6A.68.68 0 1 1 6 6.7h12a.68.68 0 1 1 0 1.36zm0 4.69H6a.68.68 0 0 1 0-1.36h12a.68.68 0 0 1 0 1.36zm0 4.69H6a.68.68 0 0 1 0-1.36h12a.68.68 0 1 1 0 1.36zm0 4.69H6a.68.68 0 0 1 0-1.36h12a.68.68 0 0 1 0 1.36z" class="a"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?